interface_frontend.py

```diff
@@ -34,8 +35,12 @@ def get_sprite(img_path, x=0, y=0):
 select_sprite = get_sprite('img/interface/png/card_cv.png')
 # Selection cursor
 cursor_sprite = get_sprite('img/interface/png/card_sl.png')
-# Test robot image
-my_robot_sprite = get_sprite('img/robots/png/mintbot.png', x=74, y=888)
+# Loading of robots images
+loaded_robots_images = {}
+for image_path in Path('./img/robots/png').iterdir():
+    loaded_robots_images[image_path.stem] = pyglet.image.load(image_path)
+
+my_robot_sprite = get_sprite('img/robots/png/hanka.png', x=74, y=888)
```

Comment:
Zde zajišťuješ, že se pokaždé vykreslí robot Hanka, neměl by se spíš vykreslit robot nalezený výše podle klíče?

Reply:
To mi řekl Petr že to tak mám udělat, tady se nastaví sice jen ten první, ale mění se právě pokaždé předtím než se sprite vykreslí na řádku 168 `    my_robot_sprite.image = loaded_robots_images[interface_state.robot.name]`

Reply:
Aha, vidíš, slepota.

Reply:
Jestli něco děláš jen proto, že jsem to řekl, tak jsem to dostatečně nevysvětlil. Nevymlouvej se na mě, a zkus to udělat líp než řeknu já :)

Reply:
Konkrétně tady je to proto, že `Sprite` potřebuje *nějaký* obrázek. V momentě vytváření Spritu ještě nevíte, jaký obrázek tam nakonec bude, takže je jedno který se tam dá.

Reply:
No tak sorry, ty jsi prostě takový Bůh a když něco řekneš, tak to platí :-D já to chápu proč to tak je, vysvětlils to dobře, jen jsem Káji špatně odpověděla :-D